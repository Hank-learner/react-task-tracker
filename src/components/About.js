import { Link } from 'react-router-dom'
import styled from "styled-components";

const StyledAbout = styled.div`
    
    margin : 10px;
    ol{
        margin:0px 30px;
    }
`;

const StyledH4 = styled.h4`
    color: ${prop => prop.heading ? 'white' : 'lime'};;
`;

const About = () => {
    return (
        <StyledAbout>
            <StyledH4 heading={false}>Version 1.0.0</StyledH4>
            <StyledH4 heading={true}>A task-note taking application</StyledH4>
            <h6>Features:</h6>
            <ol>
                <li>Add Tasks</li>
                <li>View Tasks</li>
                <li>Delete Tasks</li>
                <li>Multiple page navigation</li>
                <li>Tag Tasks as important</li>
            </ol>
            <hr className='header-break'></hr>
            <Link to='/'>Go Back</Link>
        </StyledAbout>
    )
}

export default About
