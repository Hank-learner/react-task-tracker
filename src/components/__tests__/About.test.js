import renderer from 'react-test-renderer';
import About from '../About';

// const validProps = {

// }


describe('About snapshot test', () => {
    it('renders correctly', () => {
        const tree = renderer.create(About).toJSON();
        expect(tree).toMatchSnapshot();
    });
});

// describe('render testing', () => {

//     test('renders addTask', () => {
//         const { getByText } = renderer.create(About).toJSON();
//         expect(getByText('Developed with help of Traversy Media Â© 2021', { exact: false })).not.toBeNull();
//     });
// })