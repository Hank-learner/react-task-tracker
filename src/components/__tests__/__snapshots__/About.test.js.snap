// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About snapshot test renders correctly 1`] = `null`;
